Document/0x07a-Testing-Authentication-and-Session-Management.md

```diff
@@ -305,7 +305,13 @@ Regular Expressions are often used to validate passwords. The password verificat
 
 Passwords can be set when registering accounts, changing the password or when resetting the password in a forgot password process. All of the available functions in the application that are able to change or set a password need to be identified in the source code. They should all be using the same password verification check, that is aligned with the password policy.
 
-If a frameworks is used that offers the possibility to create and enforce a password policy for all users of the application, the configuration should be checked.
+Here are different examples on how a validation can be implemented server-side:
+
+* Spring (Java) -  https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/validation/Validator.html
+* Ruby on Rails -  http://guides.rubyonrails.org/active_record_validations.html
+* PHP - http://php.net/manual/en/filter.filters.validate.php
```

Comment:
Thanks, good point. I've added the section with these three to make the doc more consistent through the subchapters (and to show how things can be done "natively") but I agree that Zxcvbn or similar are more appropriate here.